DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_static_entries.py

```diff
@@ -73,14 +72,11 @@ async def test_bridging_static_entries(testbed):
     err_msg = f"Verify that entities set to learning 'OFF' and flooding 'OFF' state.\n{out}"
     assert out[0][device_host_name]["rc"] == 0, err_msg
 
-    out = await BridgeFdb.add(
-        input_data=[{device_host_name: [
-            {"device": ports[0], "lladdr": "aa:bb:cc:dd:ee:11", "master": True, "static": True},
-            {"device": ports[1], "lladdr": "aa:bb:cc:dd:ee:12", "master": True, "static": True},
-            {"device": ports[2], "lladdr": "aa:bb:cc:dd:ee:13", "master": True, "static": True},
-            {"device": ports[3], "lladdr": "aa:bb:cc:dd:ee:14", "master": True, "static": True},
-            ]}])
-    assert out[0][device_host_name]["rc"] == 0, f"Verify that FDB static entries added.\n{out}"
+    for x in range(4):
+        out = await BridgeFdb.add(
+            input_data=[{device_host_name: [
+                {"device": ports[x], "lladdr": f"aa:bb:cc:dd:ee:1{x+1}", "master": True, "static": True}]}])
+        assert out[0][device_host_name]["rc"] == 0, f"Verify that FDB static entries added.\n{out}"
```

Comment:
Better to use list comprehension inside BridgeFdb.add() call instead of general loop which will decrease cmd execution time